docs/source/topics/con_about-presets.adoc

```diff
@@ -0,0 +1,18 @@
+[id="about-presets_{context}"]
+= About presets
+
+[role="_abstract"]
+{prod} presets represent a managed container runtime and the lower bounds of system resources required by the instance to run it.
+{prod} offers presets for {ocp} and the Podman container runtime.
+
+On {msw} and {mac}, the {prod} guided installer prompts you for your desired preset.
+On Linux, the {ocp} preset is selected by default.
+You can change this selection using the [command]`{bin} config` command before running the [command]`{bin} setup` command.
+You can change your selected preset from the system tray or command line.
```

Comment:
The system tray doesn't exist on macOS, and I believe refers to a MS Windows-specific UI element. Is there a way to specify this fact here? Maybe something like `from the system tray on {msw} or from the command line on any supported operating system`?

Reply:
Does it have another name on macOS other than "system tray"? CodeReady Containers has GUI applications which sit in the "tray" area of both Windows and macOS. The only system that does not have this functionality right now is Linux.

Reply:
Aha, it's called the Menu Bar in macOS. The question now is whether we go for complete technical accuracy or if we target the reader's understanding. I'm fairly confident that macOS users will know what "system tray" refers to, but I may be incorrect.

Reply:
Per our Slack discussion, this is OK to leave as-is